Quick/World.swift

```diff
@@ -57,12 +57,7 @@ final internal class World: NSObject {
     // MARK: Singleton Constructor
 
     private override init() {}
-    private struct Shared {
-        static let instance = World()
-    }
-    internal class func sharedWorld() -> World {
-        return Shared.instance
-    }
+    static let sharedWorld = World()
```

Comment:
I'll give it a shot. I think the old private struct way was the only way to do singletons a year ago when this code was committed. :)